### DISCUSSION TITLE
create-next-app doesn't respect user's default git branch name config

### DISCUSSION THREAD

**Discussion by lekterable** [vercel/next.js#27733] [Ideas]
### What version of Next.js are you using?

11.0.1

### What version of Node.js are you using?

16

### What browser are you using?

CLI

### What operating system are you using?

mac

### How are you deploying your application?

CLI

### Describe the Bug

create-next-app doesn't respect user's git configuration for the default branch name and always calls it `main`

### Expected Behavior

create-next-app should use user's configuration for the default branch name

### To Reproduce

run `npx create-next-app`

**Comment by lekterable** [comment]
Hi @timneutkens, what is here to discuss? 
Wouldn't respecting user's git config and using `main` as a fallback be an expected result?

**Reply by timneutkens** [reply]
This is intended behavior currently so your request is a feature request which belongs in GitHub discussions as per the issue template.
